worlds/tww/__init__.py

```diff
@@ -0,0 +1,650 @@
+            "excluded_dungeons": self.options.excluded_dungeons.value,
+            # "trap_chests": self.options.trap_chests.value,
+            "hero_mode": self.options.hero_mode.value,
+            "logic_obscurity": self.options.logic_obscurity.value,
+            "logic_precision": self.options.logic_precision.value,
+            "enable_tuner_logic": self.options.enable_tuner_logic.value,
+            "randomize_dungeon_entrances": self.options.randomize_dungeon_entrances.value,
+            "randomize_secret_cave_entrances": self.options.randomize_secret_cave_entrances.value,
+            "randomize_miniboss_entrances": self.options.randomize_miniboss_entrances.value,
+            "randomize_boss_entrances": self.options.randomize_boss_entrances.value,
+            "randomize_secret_cave_inner_entrances": self.options.randomize_secret_cave_inner_entrances.value,
+            "randomize_fairy_fountain_entrances": self.options.randomize_fairy_fountain_entrances.value,
+            "mix_entrances": self.options.mix_entrances.value,
+            "randomize_enemies": self.options.randomize_enemies.value,
+            # "randomize_music": self.options.randomize_music.value,
+            "randomize_starting_island": self.options.randomize_starting_island.value,
+            "randomize_charts": self.options.randomize_charts.value,
+            # "hoho_hints": self.options.hoho_hints.value,
+            # "fishmen_hints": self.options.fishmen_hints.value,
+            # "korl_hints": self.options.korl_hints.value,
+            # "num_item_hints": self.options.num_item_hints.value,
+            # "num_location_hints": self.options.num_location_hints.value,
+            # "num_barren_hints": self.options.num_barren_hints.value,
+            # "num_path_hints": self.options.num_path_hints.value,
+            # "prioritize_remote_hints": self.options.prioritize_remote_hints.value,
+            "swift_sail": self.options.swift_sail.value,
+            "instant_text_boxes": self.options.instant_text_boxes.value,
+            "reveal_full_sea_chart": self.options.reveal_full_sea_chart.value,
+            "add_shortcut_warps_between_dungeons": self.options.add_shortcut_warps_between_dungeons.value,
+            "skip_rematch_bosses": self.options.skip_rematch_bosses.value,
+            "remove_music": self.options.remove_music.value,
+            "death_link": self.options.death_link.value,
+        }
+
+        # Add entrances to `slot_data`. This is the same data that is written to the .aptww file.
+        all_entrance_names = [en.entrance_name for en in ALL_ENTRANCES]
+        entrances = {
+            entrance.parent_region.name: entrance.connected_region.name
+            for entrance in self.multiworld.get_entrances(self.player)
+            if entrance.parent_region is not None
+            and entrance.connected_region is not None
+            and entrance.parent_region.name in all_entrance_names
+        }
```

Comment:
I missed this when removing the intermediate entrances and entrance regions. This should be updated similarly to how `generate_output` was updated. Without this update, `entrances` is always an empty dictionary.
```suggestion
        entrances = {
            zone_entrance.entrance_name: zone_exit.unique_name for zone_entrance, zone_exit
            in self.entrances.done_entrances_to_exits.items()
        }
```

As a side-note, `all_entrance_names` would have been better as a `set` rather than a `list` here because it was only being used for `entrance.parent_region.name in all_entrance_names` checks.